xdsl/ir/affine/affine_map.py

```diff
@@ -15,6 +15,18 @@ class AffineMap:
     num_symbols: int
     results: list[AffineExpr]
 
+    @staticmethod
+    def from_constant(value: int) -> AffineMap:
+        return AffineMap(0, 0, [AffineExpr.constant(value)])
+
+    @staticmethod
+    def identity(rank: int) -> AffineMap:
+        return AffineMap(rank, 0, [AffineExpr.dimension(dim) for dim in range(rank)])
+
+    @staticmethod
+    def empty() -> AffineMap:
+        return AffineMap(0, 0, [])
+
```

Comment:
Can you make the empty method with dims = 0, syms = 0?

Reply:
you mean explicitly specifying the arguments?

Reply:
No, I meant the api. But it's fine, I can extend this later.